app/controllers/lists_controller.rb

```diff
@@ -1,13 +1,7 @@
 class ListsController < ApplicationController
-  layout :get_layout
   before_action :find_tag
```

Comment:
I can see a lot of repetition in this controller:

```ruby
@list = @tag.lists.find(params[:id])
```

I'm wondering if this controller would also benefit from a `find_list` method, the same as `ListItemsController` has.

Reply:
will do 👍

Reply:
@davidgisbey we should be able to DRY out these actions now:

- show
- edit
- confirm_destroy
- update
- edit_list_items
- update_list_items
- manage_list_item_ordering
- update_list_item_ordering

Reply:
sorry got distracted half way through fixing this 🤦 Have pushed it up!